### PR TITLE
Update 113

### PR DESCRIPTION
**Pull request by CjS77**
Tari was recently represented at the 
[International Congress of Blockchains and Applications](https://www.blockchain-congress.net/) (Blockchain '23) in 
lovely Guimarães, Portugal.

### CODE REVIEW

**Comment by CjS77** [comment]
ACK

File: _updates/2023-07-05-update-113.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+and will be published in the conference proceedings. For those who don't wish to wait, you can 
+[download the preprint now](/assets/updates/docs/tariscript.pdf).
+
+[![TariScript](/assets/updates/img/tariscript_paper_screenshot.png)](/assets/updates/docs/tariscript.pdf)
+
+The acceptance and publication of the paper is acknowledgement 
+by the broader crypto community that Tari is making significant original contributions to the space. 
+
+If you find the paper a little dense, [RFC-201](https://rfc.tari.com/RFC-0201_TariScript.html) is an 
+approachable alternative, while going into considerably greater detail than the 10-page limit of the paper allows.
```
Comment (stringhandler) [inline _updates/2023-07-05-update-113.md:28]:
```suggestion
approachable alternative, while going into considerably greater detail than the 10-page limit that the paper allows.
```

File: _updates/2023-07-05-update-113.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+If this is still over your head, then perhaps our 
+[ECC primer on Youtube](https://www.youtube.com/watch?v=69bl6dfM6jI) is
+the introduction you were looking for.
```
Comment (stringhandler) [inline _updates/2023-07-05-update-113.md:32]:
```suggestion
the introduction you are looking for.
```

**Review by stringhandler — APPROVED** [review]

File: _updates/2023-07-05-update-113.md (hunk 1 above)
Comment (CjS77) [reply _updates/2023-07-05-update-113.md:28]: Actually, I think the original is correct. "..than the paper's 10-page limit allows" might be clearer

**Pull request merged**